tests/Fixer/ControlStructure/NoBreakCommentFixerTest.php

```diff
@@ -858,6 +858,24 @@ public function provideTestFixCases()
         bar();
 }',
             ],
+            [
+                '<?php
+switch ($a) {
+    case 1:
+        throw new \Exception("");
+    case 2;
+        throw new \Exception("");
+    case 3:
+        throw new \Exception("");
+    case 4;
+        throw new \Exception("");
+    case 5:
+        throw new \Exception("");
+    case 6;
+        throw new \Exception("");
+}
+                ',
+            ],
```

Comment:
What about:

```php
<?php
switch ($a) {
    case 1:
        function () { throw new \Exception(""); };
}
```

That would need a no-break comment?

Reply:
Maybe that case already works... in which case a test would be nice. ;)

Reply:
thanks, test added :+1: